extensions/alacritty/src/utils/theme-switcher.ts

```diff
@@ -0,0 +1,84 @@
+import fs from "fs";
+import path from "path";
+import toml from "@ltd/j-toml";
+import fetch from "node-fetch";
```

Comment:
FYI the Raycast utilities package has a [`useFetch` hook](https://developers.raycast.com/utilities/react-hooks/usefetch) that might be a good fit.